packages/web-twig/src/Resources/components/Link/Link.twig

```diff
@@ -9,18 +9,18 @@
 {%- set _title = props.title | default(null) -%}
 
 {# Class names #}
-{%- set _colorClassName = _spiritClassPrefix ~ 'link-' ~ _color -%}
 {%- set _rootClassName = _spiritClassPrefix ~ 'link' -%}
-{%- set _rootDisabledClassName = _isDisabled ? _spiritClassPrefix ~ 'link-disabled' : null -%}
-{%- set _rootUnderlinedClassName = _isUnderlined ? _spiritClassPrefix ~ 'link-underlined' : null -%}
+{%- set _colorClassName = _rootClassName ~ '-' ~ _color -%}
+{%- set _rootDisabledClassName = _isDisabled ? _rootClassName ~ '-disabled' : null -%}
+{%- set _rootUnderlinedClassName = _isUnderlined ? _rootClassName ~ '-underlined' : null -%}
```

Comment:
I am sorry. I am not sure about this. IMHO this decreases readability and usability. If we keep the whole class name here as `link-underlined` (and `link-disabled`) it is better for full-text search, eye scanning of the code, and so on. We use it like this in all other components. And here we won't need the `_rootClassName` variable at all.

BTW, we keep stuff here in alphabetical order, so we would have to have an exception here.

What do you think?